src/backend/launcher.ts

```diff
@@ -1365,10 +1365,27 @@ async function runAfterLaunchScript(
  */
 async function runScriptForGame(
   gameInfo: GameInfo,
-  scriptPath: string
+  gameSettings: GameSettings,
+  scriptStage: string
 ): Promise<boolean | string> {
   return new Promise((resolve, reject) => {
-    const child = spawn(scriptPath, { cwd: gameInfo.install.install_path })
+    const scriptPath =
+      scriptStage === 'before'
+        ? gameSettings.beforeLaunchScriptPath
+        : gameSettings.afterLaunchScriptPath
```

Comment:
```ts
const scriptPath = gameSettings[`${scriptStage}LaunchScriptPath`]
```

Reply:
Thanks for the feedback, this is more simple and cleaner. Just pushed a new commit to address it.